### PR TITLE
Add main features

### PR DESCRIPTION
**Pull request by Achilles-Dev**
Added a Todo app that:
- Adds and removes todo item
- Persists todo item at local storage
- Edits added todo item

### CODE REVIEW

File: README.md
Comment (wuletawwonte) [inline README.md:56]: - [ ] Very good updating your readme, Please add the License file you mentioned here. You can use [Microverse MIT License Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)

File: src/classBased/components/TodoItem.js
Comment (wuletawwonte) [inline src/classBased/components/TodoItem.js]: - [ ] [OPTIONAL] It is better to initialize a state inside a constructor. So please add the state inside a constructor method. Because the constructor method gets called when an object is initialized based on that class.

File: src/functionBased/components/Header.js
Comment (wuletawwonte) [inline src/functionBased/components/Header.js]: - [ ] Please add the styling for this component. The tutorial has got inline styling for the header `Todos`. And, it should look like the second picture I attached here

achilles comment

completedstyles

File: src/functionBased/components/InputTodo.js
Comment (wuletawwonte) [inline src/functionBased/components/InputTodo.js]: - [ ] Please change the type of the button from `button` to `submit`. Because some old browsers won't submit the form if the type of the button is not `submit`.

File: README.md
Comment (wuletawwonte) [inline README.md]: - [ ] [OPTIONAL] Good job updating your readme but please add all the sections that exist on [Microverse template](https://github.com/microverseinc/readme-template) like `Getting Started`

**Review by wuletawwonte — CHANGES_REQUESTED** [review]
## Hi @Achilles-Dev 

Good job so far! 
There are some issues that you still need to work on before merging this branch to `dev` but you are almost there!

## To highlight :dart: 
- Your repo is professional :heavy_check_mark:
- Well structured files :heavy_check_mark: 
- No linter errors. :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please commit often, because git only takes care of your code when you commit. If you fail to commit and do something poorly thought-out, it would be difficult to trace the problem.
- [ ] [OPTIONAL] You can remove `manifest.json`, `robots.txt`or even `404.html` file. Because you don't need those files in this project. The 404.html is handled inside the project by adding a fallback route.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: src/classBased/components/TodoItem.js
Comment (Achilles-Dev) [reply src/classBased/components/TodoItem.js]: Thank you for the suggestion. However, I was following the tutorial and that is how they did it. Again the class-based component is not what I am using. It's only there because the tutorial added it to the structure. Thank you

**Review by wuletawwonte — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi @Achilles-Dev, 

Good job implementing the requested changes :+1:  . Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Some highlights for you
* Correct Gitflow practice ✔️ 
* Professional README ✔️ 
* Clean codebase ✔️ 


well-1

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**
